wiki/Usage_permissions/en.md

```diff
@@ -0,0 +1,427 @@
+---
+tags:
+  usage
+  permissions
```

Comment:
tags don't really do anything if they appear as complete words in the title already